apps/codeforafrica/src/pages/api/v1/posts.page.js

```diff
@@ -2,12 +2,25 @@ import { getPosts } from "@/codeforafrica/lib/data/utils/posts";
 import api from "@/codeforafrica/lib/payload";
 
 export default async function handler(req, res) {
-  const { primaryTag, ...other } = req.query;
+  const { primaryTag, featured, ...other } = req.query;
   if (!primaryTag) {
     return res.status(400).json({ error: "Primary Tag is required" });
   }
   try {
-    const data = await getPosts(api, other, primaryTag);
+    const data = await getPosts(
+      api,
+      {
+        ...other,
+        ...(featured && {
+          where: {
+            slug: {
+              not_equals: featured,
```

Comment:
Does this work well even when `featured` is `undefined`?

Reply:
@kilemensi I Just tested it out and it works. Also, we are making sure to add `featured` only if a featured article exists.